src/gap.h

```diff
@@ -90,6 +90,22 @@ extern Obj FuncPrint (
 
 /****************************************************************************
 **
+*F SignalBreakloop( <func> )
+**
+** Register a function which will be called when the break loop is entered
```

Comment:
I would never have guess that `SignalBreakloop` is about "registering a function". Why not let the name reflect that, e.g. `RegisterBreakloopSignalHandler` ?